csharp/README.md

```diff
@@ -0,0 +1,40 @@
+# Number Converter (C#)
+
+### Install
+
+Install [.NET Core](https://www.microsoft.com/net/core)
+
+### Clone this repository
+```bash
+$ git clone https://github.com/RappidDevelopment/NumberConverterKata.git
+$ cd NumberConverterKata/csharp
+```
+
+### Run tests via the command line
+The project's tests can be run via the command line. Depending on your operating system, different tools are available to run the unit tests.
+
+#### Windows
+Run the script `run_unit_tests_windows.bat` in the `scripts` directory
+
+```bash
+$ cd scripts
+$ run_unit_tests_windows.bat 
+```
+
+#### Linux/OSX
```

Comment:
This can be run on macOS? 😮 

I'll give this a run through tomorrow and get it setup on my machine.

Reply:
Yessir! Microsoft made .NET Core, as well as Visual Studio Community (Microsoft's IDE) available for Mac. 

I'd appreciate it if you did that and let me know if there are any fixes I need to make to the readme. 😃

Reply:
Apologies! Should be able to get to setting this up tonight